DXRenderer/HDRImageViewerRenderer.cpp

```diff
@@ -808,14 +808,23 @@ void HDRImageViewerRenderer::ComputeHdrMetadata()
 
     auto ctx = m_deviceResources->GetD2DDeviceContext();
 
+    // Reset DPI to 100% (96.0f) during histogram calculation, D2D complains
```

Comment:
Per my PR comment, I believe we no longer will hit the max texture size issue, so can you update this comment to just say that histogram rendering should always occur without DPI scaling (i.e. on raw pixels)?